src/EditorFeatures/CSharpTest/EditAndContinue/StatementMatchingTests.cs

```diff
@@ -1061,46 +1061,6 @@ int G3(int w)
             expected.AssertEqual(actual);
         }
 
-        [Fact]
```

Comment:
These tests should still pass, no?

Reply:
Do you mean tests from iteration 1? No, they should not pass. There is a match I attempted to achieve in Iteration 1:
 { "int local() { return 1; }", "int local() => 2;" }

We have 
 { "int local() { return 1; }", " => 2;" }
instead without correcting the matching algorithm.

Reply:
That is the root problem that needs fixing I think. The second local functions shouldn't be represented by an arrow expression. The matching is broken but the fix in Iteration 1 was not the right fix for that.